plugins/callback/logstash.py

```diff
@@ -43,15 +43,60 @@
             key: type
             version_added: 1.0.0
         default: ansible
+      pre_command:
+        description: Executes command before run and result put to ansible_pre_command_output field.
+        version_added: 2.0.0
+        ini:
+          - section: callback_logstash
+            key: pre_command
+        env:
+          - name: LOGSTASH_PRE_COMMAND
+      format_version:
+        description: Logging format
+        type: str
+        version_added: 2.0.0
+        ini:
+          - section: callback_logstash
+            key: format_version
+        env:
+          - name: LOGSTASH_FORMAT_VERSION
+        default: v1
+        choices:
+          - v1
+          - v2
+
+'''
+
+EXAMPLES = r'''
+ansible.cfg: >
```

Comment:
```suggestion
ansible.cfg: |
```
Otherwise you're telling a YAML parser to remove all single newlines :)